index.js

```diff
@@ -875,7 +875,7 @@ function isStreamx (stream) {
 }
 
 function defaultByteLength (data) {
-  return Buffer.isBuffer(data) ? data.length : 1024
+  return typeof data === 'object' && typeof data.byteLength === 'number' ? data.byteLength : 1024
```

Comment:
can you make a function below called isTypedArray that does the return typeof stuff so we can make this a bit easier to understand

``` js
return isTypedArray(data) ? data.byteLength : 1024
```

Reply:
Pushed https://github.com/streamxorg/streamx/pull/22/commits/16daa8d55c87d10d610ba7d1932168e77014ec6b to fix this.

Reply:
nice, forgot we need to check that its diff than null also so added that